app/src/main/java/com/github/se/gatherspot/ui/eventUI/EventDataForm.kt

```diff
@@ -84,6 +84,7 @@ import kotlinx.coroutines.runBlocking
 
 private val WIDTH = 300.dp
 private val WIDTH_2ELEM = 100.dp
+private val WIDTHWITHPICKER = WIDTH.times(0.8f)
```

Comment:
To improve readability, it is better to use SNAKE_CASE (with underscore between words)

Reply:
Yes but Android Studio sends a warning saying its not common practice for files that aren't top level

Reply:
okee